atomisticparsers/lammps/parser.py

```diff
@@ -965,14 +941,14 @@ def parse_workflow(self):
             interval_indices.append(np.arange(n_traj_split)[len(interval_indices[0]) * 3:])
 
             # calculate molecular radial distribution functions
```

Comment:
Somewhat unrelated, but it is relevant to simplifying the parser code:

We still have 2 separate implementations of calculating the rdf and msd. One within the parser and one within the workflow normalizer. I know the calculation is not being performed twice, but it would still be nice to remove this from the parser when we can. Is it already safe to do this? I feel like the workflow2 section is relatively active now, but you know better?

Reply:
I have not yet touched the msd  and rdf parts, do you prefer if we move it to MDParser?

Reply:
I will be removing msd and rdf calc in lammps, do i simply remove the tests? or do I try to transfer it to workflow norm test?

Reply:
Ah so they are alreay there. then I will simply remove the calculation part.